css/style.css

```diff
@@ -16,4 +16,132 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+#header {
+    background-color: white;
+    position: fixed;
+    max-height: 50px;
+    display: flex;
+    flex-direction: row;
+    width: 100% 
+    z-index: 9999;
+    top: 0;
+    left: 0;
+}
+
+#header-img {
+    padding: 15px 10px 30px 100px;
+    width: 25px;
+    height: 25px;
+    position: relative;
+    justify-content: flex-start;
+}
+
+#nav-bar {
```

Comment:
From previous readings, IDs can override classes (which is why I wanted to avoid using classes, but I have understood that is not correct). From today's research, I learnt IDs are easier to select with Javascript and can only be used in one element.